docs/generate_emmylua.py

```diff
@@ -14,16 +14,16 @@
     "int32_t": "integer",
     "int64_t": "integer",
     "size_t": "integer",
+    "in_port_t": "integer",
     "ImU32": "integer",
     "vector": "Array",
     "array": "Array",
```

Comment:
why those `Array` is the only one that starts from big letter? it that even important?